backend/testfiles/execution/stdlib/json.dark

```diff
@@ -136,9 +138,78 @@ Json.parse<Float> "1e3" = PACKAGE.Darklang.Stdlib.Result.Result.Ok 1000.0 // PAC
 Json.parse<Float> "-1e3" = PACKAGE.Darklang.Stdlib.Result.Result.Ok -1000.0 // PACKAGE.Darklang.Stdlib.Result.Result.Ok?
 
 // TODO: test the upper/lower bounds
+//This works but, we haven't implemented the support for numbers in the [number]e+[pow] format. we're just benefiting from .NET's support for it
+// Json.serialize<Float> 3.40282347e+38 = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+//   "3.40282347e+38"
+
+// Json.parse<Float> "3.40282347e+38" = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+//   3.40282347e+38
+
+// Json.serialize<Float> 1.17549435e-38 = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+//   "1.17549435e-38"
+
+// Json.parse<Float> "1.17549435e-38" = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+//   1.17549435e-38
+
+
+Json.parse<Float> "1.7976931348623157e+308" = PACKAGE.Darklang.Stdlib.Result.Result.Ok
+  1.7976931348623157e+308
+
 // TODO: test highly-precise numbers
+// actual 3.14159265359
+// Json.serialize<Float> 3.14159265358979323846 = PACKAGE.Darklang.Stdlib.Result.Result.Ok
```

Comment:
Not sure what to do with these tests
expected: 3.14159265358979323846
actual: 3.14159265359

Reply:
I'd look into this a bit more. Some starting points:

- Does 3.14159265358979323846 lose precision during parsing?
- What does python do in this case?

There may be some reason that json does this.